app/jobs/benchmark_regression_job.rb

```diff
@@ -0,0 +1,50 @@
+require 'uri'
+require 'net/http'
+require 'json'
+
+class BenchmarkRegressionJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(benchmark_run_id)
+
+    benchmark_run = BenchmarkRun.find_by_id(benchmark_run_id)
+
+    category = benchmark_run.result.keys[0]
+
+    current_commit = benchmark_run.initiator
+    previous_commits = Commit.where('created_at < ?', current_commit.created_at).where(repo: current_commit.repo).limit(1000)
+
+    previous_benchmark_runs = previous_commits.map {|commit| BenchmarkRun.find_by(initiator: commit, benchmark_type: benchmark_run.benchmark_type,
+                     benchmark_result_type: benchmark_run.benchmark_result_type)}
+
+    previous_benchmark_results = previous_benchmark_runs.map { |run| run.result[category].to_f }
+
+    results_param = average(previous_benchmark_results) + 2*standard_deviation(previous_benchmark_results)
+
+    if benchmark_run.result[category].to_f > results_param
+      create_issue benchmark_run
+    end
+  end
+
+  def create_issue(benchmark_run)
+    uri = URI.parse(Rails.application.secrets.github_api)
```

Comment:
Yes I suggested it because, I saw the earlier URL was pointing to Saurabh's account and there was a comment about changing it later to the actual Ruby Bench repo. 

By having ENV var, Saurabh can test locally on his repo but production repo will be Ruby Bench repo. Also it will be useful for us for testing as well :smile: